### PR TITLE
RN 2.2.1 and adding warnings for upgrade to 2.2

### PR DESCRIPTION
**Pull request by cmastr**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://support.d2iq.com/s/article/Customer-Advisory-DC-OS-2-2-0-upgrade-causes-docker-containers-to-restart-D2IQ-2020-0010 

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/mesosphere/dcos/2.1/release-notes/2.1.0/index.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ excerpt: Release notes for DC/OS 2.1.0, including Open Source attribution, and v
 ---
 Mesosphere&reg; DC/OS&trade; 2.1.0 was released on 9, June 2020.
 
+WARNING: The DC/OS 2.2 release includes an issue that changes the name of the 'libnetwork' certificate which causes its bootstrap to restart Docker and workloads, if you upgrade from 2.1.x. Because of this, we recommend you do not upgrade, and wait for a patch.
```
Comment (vinodkone) [inline pages/mesosphere/dcos/2.1/release-notes/2.1.0/index.md:13]: DC/OS 2.2.0

File: pages/mesosphere/dcos/2.1/release-notes/2.1.1/index.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ excerpt: Release notes for DC/OS 2.1.1, including Open Source attribution, and v
 ---
 DC/OS&trade; 2.1.1 was released on 27 August, 2020.
 
+WARNING: The DC/OS 2.2 release includes an issue that changes the name of the 'libnetwork' certificate which causes its bootstrap to restart Docker and workloads, if you upgrade from 2.1.x. Because of this, we recommend you do not upgrade, and wait for a patch.
```
Comment (vinodkone) [inline pages/mesosphere/dcos/2.1/release-notes/2.1.1/index.md:10]: DC/OS 2.2.0

File: pages/mesosphere/dcos/2.2/release-notes/2.2/index.md [hunk 3]
```diff
@@ -10,6 +10,8 @@ excerpt: Release notes for DC/OS 2.2, including Open Source attribution, and ver
 ---
 Mesosphere&reg; DC/OS&trade; 2.2 was released on 29, October 2020.
```
Comment (vinodkone) [inline pages/mesosphere/dcos/2.2/release-notes/2.2/index.md:11]: DC/OS 2.2.0

File: pages/mesosphere/dcos/2.2/release-notes/2.2/index.md [hunk 4]
```diff
@@ -10,6 +10,8 @@ excerpt: Release notes for DC/OS 2.2, including Open Source attribution, and ver
...
 
+WARNING: The DC/OS 2.2 release includes an issue that changes the name of the 'libnetwork' certificate which causes its bootstrap to restart Docker and workloads, if you upgrade from 2.1.x. Because of this, we recommend you do not upgrade, and wait for a patch.
```
Comment (vinodkone) [inline pages/mesosphere/dcos/2.2/release-notes/2.2/index.md:13]: DC/OS 2.2.0

File: pages/mesosphere/dcos/2.2/release-notes/2.2/index.md (hunk 3 above)
Comment (vinodkone) [reply pages/mesosphere/dcos/2.2/release-notes/2.2/index.md:11]: DC/OS 2.2.0 in "title", "navigationTitle", "excerpt:" as well

**Review by vinodkone — CHANGES_REQUESTED** [review]

**Review by orsenthil — APPROVED** [review]
LGTM.

**Comment by cmastr** [comment]
Must add DCOS 2.2.1 release notes.

**Review by vinodkone — APPROVED** [review]

**Review by cneth — APPROVED** [review]
LGTM

**Comment by cmastr** [comment]
closing in favor of #3325

**Pull request closed without merging**